articles/native-platforms/ios-swift/index.yml

```diff
@@ -16,4 +16,6 @@ articles:
   - 02-custom-login
   - 03-session-handling
   - 04-user-profile
-  - 05-linking-accounts
+  - 05-session-handling
```

Comment:
@pablolvillar Is this due to some PR? Because the file at the repo is named `05-linking-accounts` https://github.com/auth0/docs/blob/master/articles/native-platforms/ios-swift/05-linking-accounts.md